3주차/FE/sqaurelu/src/api/axios.js

```diff
@@ -0,0 +1,27 @@
+import axios from "axios";
+import CovidState from '../models/CovidState';
+
+export const fetchCovidState = async (date) => {
+    const API_SERVICE_KEY = process.env.REACT_APP_API_SERVICE_KEY;
+
+    let queryParams = '?' + encodeURIComponent('ServiceKey') + '=' + API_SERVICE_KEY; /*Service Key*/
+    queryParams += '&' + encodeURIComponent('pageNo') + '=' + encodeURIComponent('1'); /**/
+    queryParams += '&' + encodeURIComponent('numOfRows') + '=' + encodeURIComponent('10'); /**/
+    queryParams += '&' + encodeURIComponent('startCreateDt') + '=' + encodeURIComponent(date); /**/
+    queryParams += '&' + encodeURIComponent('endCreateDt') + '=' + encodeURIComponent(date); /**/
```

Comment:
이렇게 관리하는 방법도 있을 것 같아요!
```suggestion
    const queryParams = [
        `?${encodeURIComponent('ServiceKey')}=${API_SERVICE_KEY}`,
        `${encodeURIComponent('pageNo')}=${encodeURIComponent('1')}`,
        `${encodeURIComponent('numOfRows')}=${encodeURIComponent('10')}`,
        `${encodeURIComponent('startCreatedAt')}=${encodeURIComponent(date)}`,
        `${encodeURIComponent('endCreatedAt')}=${encodeURIComponent(date)}`,
    ].join("&");
```